From 619e1c461bdec508e040509f3ade6e504fdb128f Mon Sep 17 00:00:00 2001
From: Bonnie <41487185+estrogently@users.noreply.github.com>
Date: Sun, 10 Dec 2023 17:21:00 +0100
Subject: [PATCH] remove some ffmpeg external libraries

---
 packages/ffmpeg.cmake | 67 -------------------------------------------
 1 file changed, 67 deletions(-)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index e45afa8..e1334aa 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -1,47 +1,15 @@
 ExternalProject_Add(ffmpeg
     DEPENDS
         amf-headers
-        avisynth-headers
         nvcodec-headers
         bzip2
-        lame
         lcms2
-        openssl
-        libssh
-        libsrt
-        libass
-        libbluray
-        libmodplug
         libpng
-        libsoxr
-        libbs2b
-        libvpx
-        libwebp
-        libzimg
-        libmysofa
-        fontconfig
-        harfbuzz
-        opus
-        speex
-        vorbis
-        x264
-        ${ffmpeg_x265}
-        xvidcore
         libxml2
         libvpl
-        libopenmpt
         libjxl
         shaderc
         libplacebo
-        libzvbi
-        libaribcaption
-        aom
-        rav1e
-        dav1d
-        vapoursynth
-        uavs3d
-        davs2
-        rubberband
         libva
     GIT_REPOSITORY https://github.com/FFmpeg/FFmpeg.git
     SOURCE_DIR ${SOURCE_LOCATION}
@@ -60,46 +28,12 @@ ExternalProject_Add(ffmpeg
         --enable-version3
         --enable-nonfree
         --enable-postproc
-        --enable-avisynth
-        --enable-vapoursynth
-        --enable-libass
-        --enable-libbluray
-        --enable-libfreetype
-        --enable-libfribidi
-        --enable-libfontconfig
-        --enable-libharfbuzz
-        --enable-libmodplug
-        --enable-libopenmpt
-        --enable-libmp3lame
         --enable-lcms2
-        --enable-libopus
-        --enable-libsoxr
-        --enable-libspeex
-        --enable-libvorbis
-        --enable-libbs2b
-        --enable-librubberband
-        --enable-libvpx
-        --enable-libwebp
-        --enable-libx264
-        --enable-libx265
-        --enable-libaom
-        --enable-librav1e
-        --enable-libdav1d
-        --enable-libdavs2
-        --enable-libuavs3d
-        --enable-libxvid
-        --enable-libzimg
-        --enable-openssl
         --enable-libxml2
-        --enable-libmysofa
-        --enable-libssh
-        --enable-libsrt
         --enable-libvpl
         --enable-libjxl
         --enable-libplacebo
         --enable-libshaderc
-        --enable-libzvbi
-        --enable-libaribcaption
         --enable-cuda-llvm
         --enable-cuvid
         --enable-nvdec
@@ -111,7 +45,6 @@ ExternalProject_Add(ffmpeg
         --enable-vaapi
         --disable-vdpau
         --disable-videotoolbox
-        --disable-decoder=libaom_av1
         --extra-cflags='-Wno-error=int-conversion'
         "--extra-libs='${ffmpeg_extra_libs}'" # -lstdc++ / -lc++ needs by libjxl and shaderc
     BUILD_COMMAND ${MAKE}
-- 
2.42.0.windows.1

